app/controllers/api/v1/tags_controller.rb

```diff
@@ -23,6 +24,16 @@ def unfollow
     render json: @tag, serializer: REST::TagSerializer
   end
 
+  def feature
```

Comment:
Add to `api/v1/tags` request spec for this new action and the one below?

Reply:
I would like that as well, though it's not a blocker.